lstchain/mc/sensitivity.py

```diff
@@ -179,10 +179,17 @@ def calculate_sensitivity_lima(n_excesses, n_background, alpha, n_bins_energy, n
 
     """
 
+    if any(len(a) != n_bins_energy for a in (n_excesses, n_background, alpha)):
+        raise ValueError(
+            'Excess, background and alpha arrays must have the same length')
+           
+    
     stat = WStatCountsStatistic(
-        n_on=np.ones_like(n_background),
-        n_off=n_background,
-        alpha=alpha)
+        n_on = n_excesses,  
+        n_off = n_background,
+        alpha = alpha*np.ones_like(n_background)
```

Comment:
here `alpha` should be an array, so no need to multiply it by `np.ones_like(n_background)`

Also: if I'm not wrong, according to pep8, there should be no spaces between the `=` in function definitions/calls. There should, however, be spaces between operation symbols (`*`)

Reply:
Ah yes, thanks for the alpha suggestion.

And also for the spaces.